code/datums/autolathe/tools.dm

```diff
@@ -1,13 +1,11 @@
 /datum/design/autolathe/tool/crowbar
-	name = "crowbar"
```

Comment:
Why are the names removed?

Reply:
If left blank, design datum name field is auto-set to the item name. No reason to have those names in the datum in most cases.

Didn't mean to have those changes in this PR, but there is no harm in them being there.